docs/src/app/(public)/(content)/react/components/separator/demos/hero/tailwind/index.tsx

```diff
@@ -1,4 +1,3 @@
-import * as React from 'react';
```

Comment:
On https://github.com/mui/mui-public/issues/574, it feels like we should do this outside of the context of changing the bundler. One is DX for developers using the library, the other is DX for our maintainers, so once we exit the draft mode, two PRs?

Reply:
That would be the final PR where it'll be just about the bundler migration.
This is just me testing what it requires to make me the bundler work which'll then be moved to mui-public PR (all the common bits).